docs/design/chat/features_overview.md

```diff
@@ -0,0 +1,40 @@
+# 🌐 LAT-Mesh Chat Service: Your Smart Chat Assistant
+
+## 🤖 What Is LAT-Mesh?
```

Comment:
Would it be better to keep this as 'What is LAT-Mesh Chat?' since the platform will offer services outside of pure chatting?